numba/tests/test_np_functions.py

```diff
@@ -157,6 +157,10 @@ def flip(a):
     return np.flip(a)
 
 
+def logspace(start, stop, num=50):
+    return np.logspace(start, stop, num=num)
```

Comment:
Similarly to e.g. `bincount` above, both the 2 and 3 argument form should be checked so as to ensure that kwarg defaults are handle correctly.